R/gh_fill.R

```diff
@@ -0,0 +1,14 @@
+gh_fill <- function(geohashes, precision) {
+  if (length(unique(nchar(geohashes))) > 1) {
+    stop("Input Geohashes must all have the same precision level.")
+  }
+  if (any(grepl("['ailoAILO]", geohashes))) {
+    stop("Invalid Geohash; Valid characters: [0123456789bcdefghjkmnpqrstuvwxyz](any case)")
+  }
+  new_levels <- precision - nchar(geohashes[1])
+  base32 <-
```

Comment:
I would cache this to `.geohash` at build time since it's likely to come up in various places:

https://github.com/MichaelChirico/geohashTools/blob/700fa97d18da66e4132b9ff8cecd2ea6633fe7e2/R/onLoad.R#L3

```
.geohash$base32 <- ...
```